src/calls/timesheet/models/timesheet.ts

```diff
@@ -19,9 +20,7 @@ const timesheetRt = rt.Record({
   date: rt.String,
   hours: rt.Number,
   chargeableHours: rt.Number.optional(),
-  employee: rt.Record({
-    id: rt.Number,
-  }),
+  employee: employeeRt,
```

Comment:
Until further notice, we'd need to use `resourceRef` for this. Related to #63

Reply:
So, I should just close this PR, then?

Reply:
Yes, I think so.